packages/server/modules/core/graph/resolvers/versions.ts

```diff
@@ -134,18 +134,23 @@ export = {
         project
       })
 
-      let lastVersion: Version | null
+      let latestVersion: Version | null = null
+      let latestVersions: Array<Version> | null = null
       if (getTypeFromPath(info) === 'Model') {
-        lastVersion = await ctx.loaders
+        latestVersion = await ctx.loaders
           .forRegion({ db: projectDB })
           .branches.getLatestCommit.load(parent.branchId)
       } else {
-        lastVersion = await ctx.loaders
+        latestVersions = await ctx.loaders
```

Comment:
i'm a bit concerned w/ having to pull all of the project's latest versions just to answer the question of whether it is the latest version.

it may be better to have a dataloader like: streams.checkIsLatestVersion({projectId, versionId}), that would batch up these requests and then check them all w/ a single query
or if its too complicated to do 1 query for different versions of different projects, we sometimes have dataloaders scoped to specific projects (check forStream())
that way we don't actually need to pull all of the versions in memory, we only get back a bool essentially